compiler/modules/nback/number.bal

```diff
@@ -12,6 +12,34 @@ final RuntimeFunction floatToIntFunction = {
     attrs: ["nounwind", "readnone", "speculatable", "willreturn"]
 };
 
+final map<RuntimeFunction> decimalArithmeticFuncs = {};
```

Comment:
This is not the right way to do things.

- The type needs to be `readonly`
- Don't use `init`
- Initialize the variable with a call to a function that returns the map

Reply:
Changed